docs/core/tools/dotnet-nuget-add-source.md

```diff
@@ -25,6 +25,9 @@ dotnet nuget add source -h|--help
 
 The `dotnet nuget add source` command adds a new package source to your NuGet configuration files.
 
+> [!WARNING]
+> Be careful when adding multiple feeds not to introduce a [dependency confusion attack](https://aka.ms/pkg-sec-wp).
```

Comment:
```suggestion
> [!WARNING]
> When adding multiple NuGet sources, be careful not to introduce a [dependency confusion attack](https://aka.ms/pkg-sec-wp).
```

Reply:
@IEvangelist Should it be multiple package sources instead to match the name used in [nuget.config](https://docs.microsoft.com/en-us/nuget/reference/nuget-config-file#packagesources)?

Reply:
Yeah, that is fine. I updated the suggestion...